source/blog/2017-07-31-fedora-atomic-25-to-26-upgrade.md

```diff
@@ -0,0 +1,274 @@
+working. If not, then you always have the rollback command: `sudo
+rpm-ostree rollback`.
+
+**NOTE:** Over time you can see updated commands for upgrading Atomic
+          Host between releases by visiting [this](https://fedoraproject.org/wiki/Atomic_Host_upgrade)
+          wiki page.
+
+# Appendix A: Upgrading Systems with Kubernetes
+
+Fedora 25 Atomic Host ships with Kubernetes **v1.5.3**, and Fedora 26
+Atomic Host ships with Kubernetes **v1.6.7**. **Before** upgrading systems
+participating in an existing Kubernetes cluster from 25 to 26, there
+are a few configuration changes to make.
+
+## Node Servers
+
+In Kubernetes 1.6, the `--config` argument is no longer valid. If
+your `KUBELET_ARGS` in `/etc/kubernetes/kubelet` point to the manifests
+directory using the `--config` argument, then you need to change
+the argument name to `--pod-manifest-path`. Also in `KUBELET_ARGS`, you 
+need to add the argument `--cgroup-driver=systemd`.
+
+For example, if your `/etc/kubernetes/kubelet` file starts with the
+following:
+
+```nohighlight
+KUBELET_ARGS="--kubeconfig=/etc/kubernetes/kubelet.kubeconfig --config=/etc/kubernetes/manifests --cluster-dns=10.254.0.10 --cluster-domain=cluster.local"
+```
+
+Then it should be changed to be:
+
+```nohighlight
+KUBELET_ARGS="--kubeconfig=/etc/kubernetes/kubelet.kubeconfig --pod-manifest-path=/etc/kubernetes/manifests --cluster-dns=10.254.0.10 --cluster-domain=cluster.local --cgroup-driver=systemd"
+```
+
+## Master Servers
+
+### Staying With etcd2 
+
+From Kubernetes 1.5 to 1.6 upstream 
+[shifted](https://kubernetes.io/docs/tasks/administer-cluster/upgrade-1-6/)
+from using version 2 of the etcd API to version 3. The 
+[Kubernetes documentation](https://github.com/kubernetes/kubernetes/blob/93b144c/CHANGELOG.md#internal-storage-layer-1)
+instructs users to **add** two arguments to the `KUBE_API_ARGS` variable
+in the `/etc/kubernetes/apiserver` file:
+
+```nohighlight
+--storage-backend=etcd2 --storage-media-type=application/json
+```
+
+This will ensure that any pods, services or other objects stored in etcd
+will continue to be found by Kubernetes once you've completed your upgrade.
+
+### Moving To etcd3
+
+If you later wish to migrate your etcd data to the v3 api, stop your
+etcd and kube-apiserver services and run the following command to 
+migrate to etcd3:
+
+```nohighlight
+# ETCDCTL_API=3 etcdctl --endpoints https://YOUR-ETCD-IP:2379 migrate --data-dir=/var/lib/etcd
+```
+
+Then remove the `--storage-backend=etcd2` and `--storage-media-type=application/json`
+arguments from the `/etc/kubernetes/apiserver` file and restart the etcd
+and kube-apiserver services.
+
+
+# Appendix B: Fedora 25 Atomic Host Life Support
+
+We have [decided](https://pagure.io/atomic-wg/issue/303)
+to keep updating the `fedora-atomic/25/x86_64/docker-host`
+ref every day when Bodhi runs within Fedora. A new update will
+get created every day. However, we recommend you upgrade to Fedora 26,
```

Comment:
should we mention the minimal testing here?  Just because otherwise this para doesn't give folks much incentive to upgrade.

Also, we should provide/link instructions on switching to the updates ref, no?